.github/workflows/python-package.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
Curious as to why 3.6 was dropped.  Is there anything in particular in 3.6 that is causing issues?

Reply:
There was an error. I think it complained about "from __future__ import annotations" in executors/local.py. So I removed 3.6 because deciding on the exact python support matrix was not the purpose of this PR.